core/modules/contact/contact.pages.inc

```diff
@@ -76,6 +76,14 @@ function contact_site_form($form, &$form_state) {
     '#default_value' => $user->uid ? $user->mail : '',
     '#required' => TRUE,
   );
+  if (!empty($config->get('phone'))) {
+    $form['phone'] = array(
+      '#type' => 'textfield',
```

Comment:
Have you considered to use [input of type "tel"](https://github.com/backdrop/backdrop/blob/10167aae01ea19486e532871cfbaf238adebfa14/core/modules/system/system.module#L293) instead? Mobile browsers might benefit.

Reply:
Does it allow for different characters, like '+', '()', etc.?

E.g.: `+61 400 123 456` or `(02) 6654 9870`

Reply:
It doesn't restrict anything, as long as there's no "pattern" attribute - actually identical to type "text".

Here's an [example (with pattern, though)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel)

This is just a suggestion. We have it in core, so why not using it. :wink:

Reply:
Done, thanks!